.github/workflows/ok-to-test.yaml

```diff
@@ -0,0 +1,24 @@
+# File based on https://github.com/imjohnbo/ok-to-test/blob/master/.github/workflows/ok-to-test.yml
+name: Ok To Test
+
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  ok-to-test:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+
+    # Only run for PRs, not issue comments
+    if: ${{ github.event.issue.pull_request }}
+    steps:
+      - name: slash command dispatch
+        uses: peter-evans/slash-command-dispatch@a28ee6cd74d5200f99e247ebc7b365c03ae0ef3c # v3.0.1
+        with:
+          token: ${{ secrets.GH_PAT }} # we aren't allowed to use secrets.GITHUB_TOKEN here
```

Comment:
https://github.com/peter-evans/slash-command-dispatch#token

It's because the secrets.GITHUB_TOKEN doesn't have the proper permissions needed. secrets.GITHUB_TOKEN is something automatically populated by the GitHub runner (not something we create) so it's preferred to use that when possible.

Reply:
I take this back. The action documentation mimics what I said above but apparently using just the normal GH_TOKEN will work in our use case https://github.com/peter-evans/slash-command-dispatch/issues/147